### PR TITLE
163470708 Get all Political office

### PR DESCRIPTION
**Pull request by sojida**
#### What does this PR do?
Get all political office when a `GET` request is sent to `api/v1/offices`

#### Description of task to be completed?
Have this endpoint working `GET api/v1/offices`

#### How this PR can be tested
- After cloning this repo, cd into it and switch to branch
`git checkout ft-get-offices-163470708`

- run `npm run dev` to start server or `npm test` to test the endpoint

- Using Postman, test the above endpoint

#### What are the relevant pivotal tracker stories?
[#163470708](https://www.pivotaltracker.com/story/show/163470708)

#### Screenshots
- get all parties
get all offices

### CODE REVIEW

File: test/index.test.js [hunk 1]
```diff
@@ -17,3 +17,20 @@ describe('Homepage', () => {
       });
   });
 });
+
```
Comment (JCanaks) [inline test/index.test.js]: @sojida Good job but why is this test for office located within the index.test.js file? Why not have a file `office.test.js` and include all office related test there. It is cleaner to group tests into files with names relative to the test being done. This makes it easier for you to identify failing tests as your tests grow bigger

**Review by JCanaks — COMMENTED** [review]
Good job. Attend to the feedbacks and also pls include screenshots of results from Postman

File: test/index.test.js (hunk 1 above)
Comment (sojida) [reply test/index.test.js]: That true. Thanks, I'll change it

**Review by JCanaks — APPROVED** [review]
Great work. This can be merged. Take note to follow due process for rebasing

**Pull request merged**